test/unit/ansible_provider_test.rb

```diff
@@ -20,7 +20,7 @@ class AnsibleProviderTest < ActiveSupport::TestCase
 
```

Comment:
Can you try this:

```ruby
    context 'when it is using the ansible_run_host feature' do
      it 'has remote_execution_command false' do
        rex_feature = RemoteExecutionFeature.where(label: 'ansible_run_host', name: 'Run Ansible roles').first_or_create
        template_invocation.template.remote_execution_features << rex_feature
        assert_not command_options[:remote_execution_command]
      end
    end
```

I am able to reproduce the issue locally when I run `RemoteExecutionFeature.destroy_all` before the test,
fix for me was adding `name` feature and remove `let`, it's not useful anyway in this case.